### ISSUE TITLE
no connection

### ISSUE THREAD

**Issue by mike323745** [Kemmey/Kemmey-TeslaWatch-Public#384]
The app is working fine on my iPhone but not on the apple watch(series 3). The logo and Watch for Tesla is on the watch screen but login is grayed out. Does it not work with Gen 3 watches?

**Comment by Kemmey** [comment]
Hey @mike323745 

Please try restarting both your watch and your phone - the login from your phone should transfer over automatically. 

I'm a bit confused that you mention a login button on the watch - that should not be there in the current release, as login now have to happen on the phone due to requirements from The tesla login flow. 

Could you send me a screenshot of the app on your watch?

Cheers,

Kim

**Comment by mike323745** [comment]
See attached
Mike
Dr. Jonathan I.Warman MD, PC
Billing Department
Phone:516-621-3399
Fax#:516-222-5571

**Comment by mike323745** [comment]
Sure, here it is.
Mike
Dr. Jonathan I.Warman MD, PC
Billing Department
Phone:516-621-3399
Fax#:516-222-5571

**Comment by mike323745** [comment]
I have restarted both watch and iphone and there is no difference in the
watchface. Please solve the problem or I shall have to ask for the charge
back for the app.
regards,
Mike
Mike
Dr. Jonathan I.Warman MD, PC
Billing Department
Phone:516-621-3399
Fax#:516-222-5571

**Comment by Kemmey** [comment]
Hey @mike323745 

If you've successfully logged into the app on your phone, you've restarted both your watch and your phone, and the login still does not transfer over, then the general communication between your phone and your watch is the issue. 

Your login can transfer to the watch in two ways - either directly through the watch connectivity feature built in to the watch, which is the most common route. Or it can transfer over via iCloud Keychain. This usually takes a little longer but no more than a couple of minutes. 

In your case it's been days - and neither of those two have seemingly worked. As these technologies are Apple technologies, there's no real way for me to fix the issue - is this is an issue with the underlying connectivity between your phone and your watch. 

You can try to ensure handoff is enabled in Settings, General, AirPlay & Handoff:

![image](https://user-images.githubusercontent.com/4688084/128309150-a5922e57-4eba-4c9e-bbeb-ff5b1569cc39.jpeg)

And that Keychain is enabled in your iCloud settings:

![image](https://user-images.githubusercontent.com/4688084/128309101-682fa113-dbaa-41c3-b557-358bb9613ffe.jpeg)

Those are my last trouble shooting steps, unfortunately.

**Comment by mike323745** [comment]
I have no idea what happened but around 10 pm last night the watch and the
Tesla app started working together. So, it appears that you and I are
finished.
thanks for all your help
Mike
Mike
Dr. Jonathan I.Warman MD, PC
Billing Department
Phone:516-621-3399
Fax#:516-222-5571

**Comment by mike323745** [comment]
I was a little premature in stating that all was good. Although I get the
full compliment of images on the watch face they do not seem to operate the
functions that they were designed for. Any ideas?
Mike
Dr. Jonathan I.Warman MD, PC
Billing Department
Phone:516-621-3399
Fax#:516-222-5571

**Comment by Kemmey** [comment]
Can you give specific examples?

Does tapping on unlock open the trunk for instance - or does just nothing happen?

If nothing happens at all, please try opening up the app on the watch, then hold the large side button until you see the three sliders, then let go of the large button and instead hold in the crown until you're back at your watch face. Then try the command again.

**Comment by mike323745** [comment]
tapping the unlock for trunk does nothing. trunk does not open. tapping the
unlock should show that the lock unlocks but does not. This is true of all
the functions
regards,
Mike
Mike
Dr. Jonathan I.Warman MD, PC
Billing Department
Phone:516-621-3399
Fax#:516-222-5571

**Comment by mike323745** [comment]
did that. the unlock grays out and then is circled by a gray circle. I am
sending you a picture of the watch face
Mike
Dr. Jonathan I.Warman MD, PC
Billing Department
Phone:516-621-3399
Fax#:516-222-5571

**Comment by Kemmey** [comment]
Not sure I follow. 

If unlock has a light gray background, it did unlock?


Unlocked:
![image](https://user-images.githubusercontent.com/4688084/128422137-8cd88f4d-c98f-4610-8f68-85c518a04bae.png)

Locked:
![image](https://user-images.githubusercontent.com/4688084/128422159-990dcca1-3239-49dc-8bec-5131b19084b8.png)

**Comment by mike323745** [comment]
No, it doesn't look like that. It looks like what is on the watch face
image that I sent you.
Mike
Dr. Jonathan I.Warman MD, PC
Billing Department
Phone:516-621-3399
Fax#:516-222-5571

**Comment by Kemmey** [comment]
@mike323745 I have not received an email with the screenshot - GitHub replies does not support attachments. You'll have to either add the image along with your message from the GitHub site - or send me at my email address for the app as before at waft@kimhansen.dk - I'm sorry about the confusion with attachments.
